o1vm/src/interpreters/mips/constraints.rs

```diff
@@ -72,6 +73,16 @@ impl<Fp: Field> InterpreterEnv for Env<Fp> {
         self.constraints.push(assert_equals_zero)
     }
 
+    fn activate_selector(&mut self, selector_idx: usize) {
+        let mut acc : Self::Variable = -Expr::Atom(ExprInner::Constant(ConstantExpr::Atom(ConstantExprInner::Constant(kimchi::circuits::expr::ConstantTerm::<Fp>::Literal(Fp::one())))));
```

Comment:
Is there a less verbose way of writing this?  it's just `-1`.

Reply:
There is a From<u64> for expr